utilities/physics_tendency_updaters.meta

```diff
@@ -0,0 +1,303 @@
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ cpair ]
+  standard_name = specific_heat_of_dry_air_at_constant_pressure
+  units = J kg-1 K-1
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#####################################################################
+[ccpp-table-properties]
+  name = apply_tendency_of_air_temperature
+  type = scheme
+[ccpp-arg-table]
+  name = apply_tendency_of_air_temperature_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of columns
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ t_tend ]
+  standard_name = tendency_of_air_temperature
+  units = K s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ temp ]
+  standard_name = air_temperature
+  state_variable = true
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dtdt ]
+  standard_name = tendency_of_air_temperature_due_to_model_physics
+  units = K s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ cpair ]
+  standard_name = specific_heat_of_dry_air_at_constant_pressure
+  units = J kg-1 K-1
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#########################################################
+[ccpp-table-properties]
+  name = update_dry_static_energy
+  type = scheme
+
+[ccpp-arg-table]
+  name = update_dry_static_energy_init
+  type = scheme
+[ gravit_in ]
+  standard_name = gravitational_acceleration
+  units = m s-2
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+
+[ccpp-arg-table]
+  name = update_dry_static_energy_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of vertical levels
```

Comment:
Closer, but still no cigar...  Wrong long name for this standard name

Reply:
I will fix these (also those I copied from in state_converters, kessler, and kessler_update).